### ISSUE TITLE
Unable to build DB browser on Ubuntu 18.10+

### ISSUE THREAD

**Issue by deepsidhu1313** [sqlitebrowser/sqlitebrowser#1639]
Details for the issue
--------------------

#### What did you do?

Tried to build from latest commit on Ubuntu 18.10.

#### What did you expect to see?

Debian Packages

#### What did you see instead?

[Error](https://launchpadlibrarian.net/399744149/buildlog_ubuntu-cosmic-i386.sqlitebrowser_3.10.100ubuntu1-0~2238~201811302344~ubuntu18.10.1_BUILDING.txt.gz)
 
Useful extra information
-------------------------

The info below often helps, please fill it out if you're able to. :)

#### What operating system are you using?

- [ ] Windows: ( _version:_ ___ )
- [x] Linux: ( _Ubuntu:_ 18.10 )
- [ ] Mac OS: ( _version:_ ___ )
- [ ] Other: ___

#### What is your DB4S version?

- [ ] 3.11.0-alpha1 or 3.11.0-beta*
- [ ] 3.10.1
- [x] Other: latest

#### Did you also

- [ ] Try out the latest nightly version: https://github.com/sqlitebrowser/sqlitebrowser#nightly-builds
- [x] Search for an existing similar issue: https://github.com/sqlitebrowser/sqlitebrowser/issues?utf8=%E2%9C%93&q=is%3Aissue%20

I am creating this issue, so that we don't forget about it and if anyone other than me gets chance to look into, they can (people are welcomed).

Thanks !!

**Comment by justinclift** [comment]
Yeah.  The latest version of CMake known to work is version 3.10.3. :frowning:

**Comment by mgrojo** [comment]
I think I'm going to finally update to Ubuntu 18.04. I was delaying it in fear of being unable to build DB4S, but I'd like now to help with #1361. Besides Ubuntu 16.04 is feeling now a bit oldish :smile:

**Comment by FriedrichFroebel** [comment]
@mgrojo Ubuntu 18.04 seems to use CMake 3.10.2 which is not affected by the problem (at least it is like this on my Ubuntu system). The problematic CMake version seems to be shipped from Ubuntu 18.10 on at the moment.

**Comment by mgrojo** [comment]
@FriedrichFroebel Thank you. I misread the issue. But in any case my fear was for whatever might happen in the update.

Just for helping, is there a simple way to get a newer CMake version in Ubuntu 18.04?

**Comment by FriedrichFroebel** [comment]
@mgrojo Most likely https://askubuntu.com/a/865294/ will work.
